code/modules/sprite_accessories/_accessory_hair.dm

```diff
@@ -842,4 +842,7 @@
 
 /datum/sprite_accessory/hair/kajam
 	name = "Kajam"
-	icon_state = "hair_kajam"
\ No newline at end of file
+	icon_state = "hair_kajam"
+/datum/sprite_accessory/hair/hair_wavyshoulder_down_s
+	name = "Short Wavy"
+	icon_state = "hair_hair_wavyshoulder_down_s"
```

Comment:
The `hair_hair_wavyshoulder_down_s_s` icon_state does not exist, you probably meant `hair_wavyshoulder_down_s`.

Reply:
Ah. I absolutely did. My mistake!